python/lsst/ts/envsensors/device/mock_device.py

```diff
@@ -0,0 +1,122 @@
+# This file is part of ts_envsensors.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MockDevice"]
+
+import asyncio
+import logging
+import random
+from typing import Callable
+
+from .base_device import BaseDevice, DELIMITER, TERMINATOR
+from ..constants import DISCONNECTED_VALUE, Temperature
+from ..sensor import BaseSensor
+
+
+class MockDevice(BaseDevice):
+    """Mock Sensor Device.
+
+    Parameters:
+    -----------
+    name: `str`
+        The name of the device.
+    device_id: `str`
+        The hardware device ID to connect to.
+    sensor: `BaseSensor`
+        The sensor that produces the telemetry.
+    callback_func : `Callable`
+        Callback function to receive the telemetry.
+    _disconnected_channel: `int`, optional
```

Comment:
Please add the `log` parameter and fix `_disconnected_channel` -> `disconnected_channel`/

Also what do you mean by "channels number"? Should that be "channel number"?

Reply:
Yes, `channels number` should be `channel number`. All done.